src/pages/home/report/ReportActionItem.js

```diff
@@ -449,6 +449,7 @@ function ReportActionItem(props) {
             ref={popoverAnchorRef}
             onPressIn={() => props.isSmallScreenWidth && DeviceCapabilities.canUseTouchScreen() && ControlSelection.block()}
             onPressOut={() => ControlSelection.unblock()}
+            onPress={() => {}}
```

Comment:
Consider making the onPress function optional?

Reply:
It's already optional, but we could use a better default

Reply:
Instead of passing empty function here. Can we check if onPress is present and call in the PressableWithFeedback component 

https://github.com/Expensify/App/blob/561297e950d94c3a9e85d1dfb1ff0edfc586ac0e/src/components/Pressable/PressableWithFeedback.js#L77-L80

Seems we are doing the same for onPressIn and onPressOut

https://github.com/Expensify/App/blob/561297e950d94c3a9e85d1dfb1ff0edfc586ac0e/src/components/Pressable/PressableWithFeedback.js#L69-L76

Reply:
updated